add custom http headers (#110)
demisto/demisto-py: CHANGELOG.md

[PyPI History][1]
[1]: https://pypi.org/project/demisto-py/#history

#3.2.9

> * Added the ability to use custom headers for each request.

Reviewer: ```suggestion
* Added the ability to add custom request headers.
```
